R/dashboard_modules/04-learner_characteristics.R

```diff
@@ -82,9 +82,10 @@ learner_characteristics_ui <- function(id) {
         shinyGovstyle::radio_button_Input(
           inputId = NS(id, "file_type"),
           label = h2("Choose download file format"),
-          hint_label = "The XLSX format is designed for use in Microsoft Excel",
-          choices = c("CSV (Up to 11.61 MB)", "XLSX (Up to 2.91 MB)"),
-          selected = "CSV (Up to 11.61 MB)"
+          hint_label = "This will download data related to all providers for all years.
+          The XLSX format is designed for use in Microsoft Excel",
+          choices = c("CSV (Up to 13.51 MB)", "XLSX (Up to 3.23 MB)"),
```

Comment:
If the file for this is always just the whole thing (looks like you've changed it to that?) we could have:

```
choices = c("CSV (13.51 MB)", "XLSX ( 3.23 MB)"
```